### DISCUSSION TITLE
Loco positioning system inaccurate measurements

### DISCUSSION THREAD

**Discussion by manuelmaiorano** [bitcraze/discussions#973] [Q&A]
Hi, the position measurements of my loco positioning system with a 8 anchor setup seems to be inaccurate especially in the z direction,  i have updated the firmware of the anchors as well as the loco positioning deck, and the position measurements of the anchors i took are correct. I used this setup to position the anchors:

![top](https://github.com/bitcraze/crazyflie-firmware/assets/114679700/095fc6ac-26ac-4769-95f3-38cb6d3a27c7)
![bottom](https://github.com/bitcraze/crazyflie-firmware/assets/114679700/3b249d81-f20f-4371-be43-9c5c3a118499)

**Comment by knmcguire** [comment]
This offset is due to a variety of factors and in general the loco positioning's z-axis estimate is worse. As this is related to too many factors I'm linking more discussions that were based on this:

- https://github.com/orgs/bitcraze/discussions/737
- https://github.com/orgs/bitcraze/discussions/638
- https://github.com/orgs/bitcraze/discussions/276

You can also search for more discussions about this in the old forum https://forum.bitcraze.io/

Let us know if you have anymore questions.
